{"uid":"e64c9ee2d1cce4d","name":"test_charm_garbage_ignorance","fullName":"tests.integration.ha_tests.test_smoke#test_charm_garbage_ignorance","historyId":"b8dac20e94bae8d55ddc12bdeb3d4c2e","time":{"start":1721008558923,"stop":1721008558923,"duration":0},"description":"Test charm deploy in dirty environment with garbage storage.","descriptionHtml":"<p>Test charm deploy in dirty environment with garbage storage.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 85, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 844, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 873, in pytest_runtest_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 82, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 159, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 517, in setup\n    raise exc\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1635, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 686, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 521, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 606, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1076, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 228, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1721008162428,"stop":1721008162428,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1721008162428,"stop":1721008182378,"duration":19950},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1721008162428,"stop":1721008162428,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1721008182378,"stop":1721008182609,"duration":231},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1721008558923,"stop":1721008558923,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 228, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-arm64-postgresql-operator/test-runners-2-is-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"charm::finalizer","time":{"start":1721008559615,"stop":1721008559615,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1721008559616,"stop":1721008587405,"duration":27789},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1721008587406,"stop":1721008587406,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1721008587406,"stop":1721008587406,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1721008587406,"stop":1721008587406,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"openstack-arm-large-20-b4f17d38dba45b71342b0b82"},{"name":"thread","value":"27510-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"9d83a45fe72b3a4","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":6,"skipped":4,"passed":12,"unknown":12,"total":34},"items":[{"uid":"50f4af1394501b77","reportUrl":"../109//#testresult/50f4af1394501b77","status":"passed","time":{"start":1720921334074,"stop":1720921754829,"duration":420755}},{"uid":"16f441932049cbd5","reportUrl":"../108//#testresult/16f441932049cbd5","status":"unknown","time":{}},{"uid":"e489a5610261b8b0","reportUrl":"../107//#testresult/e489a5610261b8b0","status":"passed","time":{"start":1720748841204,"stop":1720749276591,"duration":435387}},{"uid":"8b1de6f58042d3c6","reportUrl":"../106//#testresult/8b1de6f58042d3c6","status":"unknown","time":{}},{"uid":"160d825fe2c4e6e9","reportUrl":"../105//#testresult/160d825fe2c4e6e9","status":"unknown","time":{}},{"uid":"2fe84930c47922ad","reportUrl":"../104//#testresult/2fe84930c47922ad","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-test-app/1 [allocating] waiting: waiting for machine","time":{"start":1720489462129,"stop":1720491464167,"duration":2002038}},{"uid":"afdef0f306098b3d","reportUrl":"../103//#testresult/afdef0f306098b3d","status":"passed","time":{"start":1720403176278,"stop":1720403708986,"duration":532708}},{"uid":"c617ab16373e6327","reportUrl":"../102//#testresult/c617ab16373e6327","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1720316607164,"stop":1720316607164,"duration":0}},{"uid":"c71e5daae209760b","reportUrl":"../101//#testresult/c71e5daae209760b","status":"passed","time":{"start":1720230232326,"stop":1720230516938,"duration":284612}},{"uid":"550f950dccced36a","reportUrl":"../100//#testresult/550f950dccced36a","status":"passed","time":{"start":1720144124532,"stop":1720144688133,"duration":563601}},{"uid":"fe225ae0b22f8421","reportUrl":"../99//#testresult/fe225ae0b22f8421","status":"unknown","time":{}},{"uid":"8a941efc63bc9946","reportUrl":"../98//#testresult/8a941efc63bc9946","status":"unknown","time":{}},{"uid":"46c23ccc40a19587","reportUrl":"../97//#testresult/46c23ccc40a19587","status":"passed","time":{"start":1719884564371,"stop":1719884991943,"duration":427572}},{"uid":"2a826f56f6e2d76","reportUrl":"../96//#testresult/2a826f56f6e2d76","status":"passed","time":{"start":1719798384271,"stop":1719798729275,"duration":345004}},{"uid":"3c23c89609875bfc","reportUrl":"../95//#testresult/3c23c89609875bfc","status":"passed","time":{"start":1719711913514,"stop":1719712323924,"duration":410410}},{"uid":"1e9ccaa29e93a6ac","reportUrl":"../94//#testresult/1e9ccaa29e93a6ac","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql-test-app/1","time":{"start":1719625709608,"stop":1719626200407,"duration":490799}},{"uid":"feb5506fb0452160","reportUrl":"../93//#testresult/feb5506fb0452160","status":"broken","statusDetails":"psycopg2.OperationalError: connection to server at \"10.213.130.9\", port 5432 failed: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.","time":{"start":1719539463801,"stop":1719539898384,"duration":434583}},{"uid":"10ee21de71f2be2f","reportUrl":"../92//#testresult/10ee21de71f2be2f","status":"broken","statusDetails":"psycopg2.OperationalError: connection to server at \"10.14.73.23\", port 5432 failed: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.","time":{"start":1719452469449,"stop":1719452856589,"duration":387140}},{"uid":"ffde663b9b4dfc17","reportUrl":"../90//#testresult/ffde663b9b4dfc17","status":"broken","statusDetails":"websockets.exceptions.ConnectionClosedError: no close frame received or sent","time":{"start":1719366387209,"stop":1719367234891,"duration":847682}},{"uid":"bc8d0956d87ef685","reportUrl":"../89//#testresult/bc8d0956d87ef685","status":"passed","time":{"start":1719279784367,"stop":1719280351914,"duration":567547}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"e64c9ee2d1cce4d.json","parameterValues":[]}